toolkit/templates/button.html

```diff
@@ -1 +1 @@
-<input type="submit" class="button-{{ type }}" {% if name %}name="{{ name }}"{% endif %} value="{{ label }}" />
+<input type="submit" class="button-{{ type }}{% if advice is defined %}-with-advice{% endif %}" {% if name %}name="{{ name }}"{% endif %} value="{{ label }}" />
```

Comment:
It turns out that `is defined` is not supported by Nunjucks, which makes it hard to reuse our templates in the prototype, so we're trying to phase it out (slowly). `{% if advice %}` does pretty much the same thing in this case, so I'd suggest we go with that.